apps/bot_manager/lib/protobuf/messages.pb.ex

```diff
@@ -299,7 +308,14 @@ defmodule BotManager.Protobuf.GameState do
   field(:status, 11, type: BotManager.Protobuf.GameStatus, enum: true)
   field(:start_game_timestamp, 12, type: :int64, json_name: "startGameTimestamp")
   field(:items, 13, repeated: true, type: BotManager.Protobuf.GameState.ItemsEntry, map: true)
-  field(:pools, 14, repeated: true, type: BotManager.Protobuf.GameState.PoolsEntry, map: true)
+
+  field(:obstacles, 14,
+    repeated: true,
+    type: BotManager.Protobuf.GameState.ObstaclesEntry,
+    map: true
+  )
+
+  field(:pools, 15, repeated: true, type: BotManager.Protobuf.GameState.PoolsEntry, map: true)
```

Comment:
Hmmm you're pushing :pools from 14th to 15th position. Isn't that what we should avoid?

Reply:
Answered in Slack. What happened is that the protobuf files generated by messages.proto file were outdated.
Just ran `make generate-protos` then pushed.